app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -185,6 +185,8 @@ public void onReceive(Context context, Intent intent) {
     // Experimental support for rfduino from Tomasz Stachowicz
     private static volatile BluetoothGattCharacteristic mCharacteristicSend;
     private byte[] lastdata = null;
+    private static BluetoothManager myBluetoothManager = null;//for display in system status
```

Comment:
For readability it would be ok to use comments either before line or after line. If you choose to use after line, a space between ";" and "//" helps. But this is only a suggestion.

Reply:
done, thanks